test/corpus/expressions

```diff
@@ -38,3 +38,27 @@ f(1) + f(1);
             (constant
               (integer)))))))
   (nl))
+
+================================================================================
+Global IDs
+================================================================================
+
+global x: ::X;
+global x: GLOBAL::X;
```

Comment:
This version is actually disallowed now (Zeek reports an error).

Reply:
IMO this fine as we still want to support parsing this. There is nothing special about `GLOBAL` as far as parsing is concerned.